### ISSUE TITLE
Cannot Set ID Value on Field to Associate with Label.

### ISSUE THREAD

**Issue by dayander** [erikras/redux-form#4208]
To support users of Assistive Technology, an ID needs to match the label element attribute for (htmlFor). Currently without setting an ID for input fields breaks [WCAG 3.3.2 Labels or Instructions.](https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions.html)

Currently you can nest a Field component inside of an input tag, but by doing this you are implicatly implying the label for the input field and this is not best practice.

**Comment by gustavohenke** [comment]
And what are you reporting, exactly? Are you not able to set IDs?
